lib/apiservers/portlayer/swagger.yml

```diff
@@ -440,6 +440,30 @@ paths:
           description: "Error"
           schema:
             $ref: "#/definitions/Error"
+    delete:
+      description: "Remove a container from existence"
+      operationId: ContainerRemove
+      tags: ["containers"]
+      consumes:
+        - application/octet-stream
```

Comment:
I think we can, @sflxn asked me to add that in with octet-stream. So it should be ok either way.